samples/apps/copilot-chat-app/documentuploadapp/appsettings.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+    "ServiceConfig": {
```

Comment:
I believe the default configuration for the SK service will be to use an API key. If so, let's provide default the access to use that.

Reply:
Will add in an upcoming change.